app/Nova/Feedback.php

```diff
@@ -55,23 +65,26 @@ class Feedback extends Resource
     public function fields(Request $request)
     {
         return [
-            ID::make()->sortable(),
-
-            BelongsTo::make('Subject', 'account', 'App\Nova\Account'),
-
-            BelongsTo::make('Submitted By', 'submitter', 'App\Nova\Account'),
+            ID::make()->sortable()->hideFromDetail(),
 
             Text::make('Feedback Form', function () {
                 return $this->form->name;
             }),
 
+            BelongsTo::make('Subject', 'account', 'App\Nova\Account'),
+
+            BelongsTo::make('Submitted By', 'submitter', 'App\Nova\Account')
+                ->canSeeWhen("seeSubmitter", $this),
+
+            DateTime::make('Submitted At', 'created_at')->onlyOnDetail()->format('Do MMMM YYYY HH:mm'),
```

Comment:
Would be a good idea to move the date formatting string to a static class member to use elsewhere.

`
public static class DateFormat
{
    public const string DefaultFormat = "Do MMMM YYYY HH:mm";
}
`

`
DateTime::make('Submitted At', 'created_at')->onlyOnDetail()->format(DateFormat.DefaultFormat),
`

Reply:
We're driving this with Laravel Nova so it's a little limited in that regard. My view is it would be better to continue using the Nova supported `->format` method, but interested to hear @AxonC's view!

Reply:
I'm easy on this one - we are constrained by the software here, but defining it as a const within that resource, or a base resource wouldn't be a bad thing.